Source/courses.py

```diff
@@ -0,0 +1,75 @@
+# Author: Isa Blancett
+# Project: Interactive Programming
+# Date: 10.30.2017
+# Description: Class for reading in a csv file containing course information and
+# creating and instance for each course
+# Acknowledgements: Script structure is modeled after Transcriptase's Game
+# repository on Git
+
+import csv
+
+class Courses(object):
+    """ class for the different course options the user can choose from
+
+    contains:
+    label - 4 letter id
+    course - full title
+    description - in-depth summary of course
+    max - total number of levels
+    reqs - dependencies, '%s%s%s' % (effected level, needed course, needed level)
+    lvl - current level
+    order - order picked by user
+    """
+
+    def setup(self, config):
+
+        self.config = config
+        self.label = config['label']
+        self.name = config['course']
+        self.description = config['description']
+        self.max = config['max']
+        self.reqs = config['dependencies']
+        self.range = config['xrange']
+        self.lvl = 0
+        self.order = 0
+
+
+
+def create_course(config):
+    """ creates a single instance of a course given a config dictionary
+
+    returns: single course instance
+    """
+
+    new_course = Courses()
+    new_course.setup(config)
+    return(new_course)
```

Comment:
`return` is a statement, not a function; the parentheses aren't necessary (or idiomatic).